### PR TITLE
Change the version number to allow packages to use the same PPA

### PR DESCRIPTION
**Pull request by mckees**
This changes the format of the versions from `xxxx-0ubuntu1~ppa1` to `xxxx-0ubuntu1~24.10~ppa1` so that the noble and oracular versions do not collide in the PPA.

The version is simply overwritten because the new version will technically be older than the old version ("p" > "2"), so we should start over at the beginning. Since this is only in our testing PPA so far, we don't need to worry about breaking anyone's updates.

### CODE REVIEW

**Comment by mckees** [comment]
@hector-cao @phckopper same as the other last PR, just for oracular. Also, I don't seem to have merge rights even after an approval, so maybe we should add that (or just get in the habit of merging after approving)

File: debian/changelog
Comment (hector-cao) [inline debian/changelog:1]: @mckees  Since you are squashing the changelog, can you also remove `Cleanup comments and extra directories` from the changelog and also set you as author with the right date (right now Pedro is the author with the old date)

**Comment by hector-cao** [comment]
> @hector-cao @phckopper same as the other last PR, just for oracular. Also, I don't seem to have merge rights even after an approval, so maybe we should add that (or just get in the habit of merging after approving)

Gave you the admin rights, i m fine with manual merge but not against automatic merging even if i think that manual merging is safer

**Review by hector-cao — CHANGES_REQUESTED** [review]

File: debian/changelog
Comment (mckees) [reply debian/changelog:1]: Alright, I've updated that now

**Review by hector-cao — APPROVED** [review]
LGTM !

**Pull request merged**
